### PR TITLE
Push meeting minutes to news part of the website

### PR DESCRIPTION
**Pull request by likeajumprope**
This commit will push the edited meeting minutes of August and September 2023 to the News part of the website in markdown format.

### CODE REVIEW

File: _posts/2023-09-30-Minutes-of_SCM-29.md [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+- JS
+
+## Fiscal Sponsorship 
+- October 31st is the critical deadline for the end of the funding of the RSE steering committe to QCIF. The steering committee is currently wokring on extedning financial sponsorship through Open Life Sciences.
+- M ACTION: to send around both documents for OLS to the steering committee this week.
+- Secretary will submit invoices to get paid in advance for hours until the end of the year before the October shutdown. 
+
+
+## Debrief Unconference
+- Despite an agreement on the great success of the unconference, the steering committee concluded that the sustainable continuation of the unconference might incldue hiring a person dedicated to planning the next unconference.
```
Comment (rowlandm) [inline _posts/2023-09-30-Minutes-of_SCM-29.md:28]: Include spelled wrong...

**Review by rowlandm — COMMENTED** [review]
All looks good. Minor spelling mistake.

File: _posts/2023-08-31-Minutes-of-SCM-28.md [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+
+
+
+## RSE UK submission 
+
+- Johanna is going and going to give a 2 min talk on RSEAUNZ in the leaders meeting
+
+## Getting new SC members 
+- we need new steering committee members 
+- we need an advisory panel, and we regular members
```
Comment (manodeep) [inline _posts/2023-08-31-Minutes-of-SCM-28.md:60]: This sentence needs to be fixed

File: _posts/2023-09-30-Minutes-of_SCM-29.md [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+- From the RSEAA meeting, could we identify and recruit some new SC members? 
+- All: Please add your recommendations, and let's identify who would be good to have onboard
+
+## New meeting time? 
+
+- Rescheduling of meeting time to ~2 pm Melbourne time to allow attendence for steering committee memmbers currently residing overseas.
+
+## RSE team leader workshop
+- Maybe give a workshop to leaders of the community (management for RSE's) (Rowland). This could also be a part of the unconference.
+- another potential option would be an RSE team leader AMA ask me anything.
```
Comment (manodeep) [inline _posts/2023-09-30-Minutes-of_SCM-29.md:73]: One of "Ask Me Anything" or "AMA" needs to be in "()"

**Review by manodeep — CHANGES_REQUESTED** [review]

**Comment by manodeep** [comment]
I swapped @nooriyahp for @Winterstorm-j

File: _posts/2023-08-31-Minutes-of-SCM-28.md (hunk 2 above)
Comment (rowlandm) [reply _posts/2023-08-31-Minutes-of-SCM-28.md:60]: we need an advisory panel, and regular members

woudl that be ok?

**Review by Winterstorm-j — APPROVED** [review]

File: _posts/2023-08-31-Minutes-of-SCM-28.md (hunk 2 above)
Comment (likeajumprope) [reply _posts/2023-08-31-Minutes-of-SCM-28.md]: I pushed two commits to fix this.

File: _posts/2023-09-30-Minutes-of_SCM-29.md (hunk 1 above)
Comment (likeajumprope) [reply _posts/2023-09-30-Minutes-of_SCM-29.md]: I pushed two commits to fix this.

File: _posts/2023-09-30-Minutes-of_SCM-29.md (hunk 3 above)
Comment (likeajumprope) [reply _posts/2023-09-30-Minutes-of_SCM-29.md]: I pushed two commits to fix this.

**Review by rowlandm — APPROVED** [review]
Ok to merge

**Review by rowlandm — APPROVED** [review]

**Pull request merged**